examples_tests/06.MeshLoaders/main.cpp

```diff
@@ -94,6 +95,19 @@ class SimpleCallBack : public video::IShaderConstantSetCallBack
 
 int main()
 {
+	core::quaternion q(1, 2, 3, 4);
```

Comment:
at least normalize it before doing stuff with it!!!!
(all library functions will assume normalized quaternion input in order to be faster)